MagnaLorem/CampusAlly
# Userclass updatedelete

All backend functionality for the user's classes is now complete. The user now has a persistent list of classes being updated in the database. The user should be able to add/update courses and delete courses. The classes to choose from now only load when searching for a class (this cuts down on the load times for ng-repeat). I have also made some html and css adjustments to make the page easier to use.

To test:
(1) Add courses by searching. Make sure the plus turns to a check mark when selecting the class.
(2) Delete courses from your list of classes. Once you search for that same class again it should have a plus next to it again meaning you can add it again.
